test/cases/measure/data/testdata/service_instance_cpm_minute_data.json

```diff
@@ -155,7 +155,7 @@
           },
           {
             "str": {
-              "value": "svc_3"
```

Comment:
You should have mocked a `Null` instead of an empty string. Could you cover two kinds of `Null`:

1. A null type by `{"null":0}`
2. A tag array with fewer elements by removing the last value from these three elements.

Reply:
I'm not sure if we can do this in this case.

ServiceID is not only the groupBy tag but also the `SeriesID` of the source measure. It cannot be NULL?

Reply:
It could be. groupBy tag could be any tags defined by a measure more than tags in the entity. ServiceID is a scenario from OAP that should not limit the scope of test cases.

Reply:
Another case is added in order to verify the NULL group